packages/loot-core/src/client/actions/account.ts

```diff
@@ -113,67 +164,73 @@ export function syncAccounts(id?: string) {
 
     dispatch(setAccountsSyncing(accountIdsToSync));
 
+    const accountsData = await send('accounts-get');
+    const simpleFinAccounts = accountsData.filter(
+      a => a.account_sync_source === 'simpleFin',
+    );
+
     let isSyncSuccess = false;
+    const newTransactions = [];
+    const matchedTransactions = [];
+    const updatedAccounts = [];
+
+    if (batchSync && simpleFinAccounts.length > 0) {
+      console.log('Using SimpleFin batch sync');
+
+      const res = await send('simplefin-batch-sync', {
+        ids: simpleFinAccounts.map(a => a.id),
+      });
+
+      for (const account of res) {
+        const success = handleSyncResponse(
+          account.accountId,
+          account.res,
+          dispatch,
+          newTransactions,
+          matchedTransactions,
+          updatedAccounts,
+        );
+        if (success) isSyncSuccess = true;
+      }
```

Comment:
This block can't really fail (well, if it does it'll be for other reasons and captured elsewhere), the functions it calls pass back errors as objects instead of throwing them so as not to stop successful accounts syncing